be/src/storage/task/engine_clone_task.cpp

```diff
@@ -316,7 +352,8 @@ Status EngineCloneTask::_clone_copy(DataDir& data_dir, const string& local_data_
 
 Status EngineCloneTask::_make_snapshot(const std::string& ip, int port, TTableId tablet_id, TSchemaHash schema_hash,
                                        int timeout_s, const std::vector<Version>* missed_versions,
-                                       std::string* snapshot_path, int32_t* snapshot_format) {
+                                       const std::vector<int64_t>* missing_version_ranges, std::string* snapshot_path,
```

Comment:
how about using reference?

Reply:
other arguments use pointer, I think better keep the convention here, we can use reference in new code